src/main/java/org/miracum/etl/fhirtoomop/mapper/ObservationMapper.java

```diff
@@ -142,6 +142,11 @@ public OmopModelWrapper map(Observation srcObservation, boolean isDeleted) {
       return null;
     }
 
+    if (!srcObservation.hasCategory() || srcObservation.getCategory().isEmpty()) {
```

Comment:
We're skipping observations without categories?

Reply:
Oh, you just moved the check

Reply:
Yeah, it was throwing an NPE if it returned null so I just made it "required" properly.